### ISSUE TITLE
Can't upload videos longer then 1min or larger then 100+MB

### ISSUE THREAD

**Issue by al73rus** [immich-app/immich#13572]
### The bug

I have some videos on my phone which i can't upload. They are 1+min long or 100+MB in size.
Mobile app doen't give an error just doesn't upload it and never try again, on webpage it just ends up with an error (no info on what error, just can't upload).
I tried downloading test vid longer/larger then i mentioned on desktop and still didn't work.
All vid bellow that level are uploading without a problem.
Didn't change any settings or upgraded storage etc. It started around 2-3 weeks ago (maybe from version 116 can't point it directly to what wersion)

### The OS that Immich Server is running on

Unraid 7 beta 3 

### Version of Immich Server

Latest

### Version of Immich Mobile App

Latest

### Platform with the issue

- [ ] Server
- [X] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
Unraid docker
```

### Your .env content

```Shell
None
```

### Reproduction steps

1. Open app/page
2. Select a video longer than 1+min ir larger than 100+MB
3. Upload and get ann error on webpage, on the app there is no info.
...

### Relevant log output

```shell
There is no error after failed atempt.got only info with the asset uploading and then nothing
```

**Comment by Mraedis** [comment]
Are you using cloudflare proxy? It has a 100MB limit.

**Comment by al73rus** [comment]
Yes i am, but still tried to connect to the app localy in the app with the local adress at home and still i didn't upload it.

**Comment by al73rus** [comment]
Is there a way to make that limit bigger? I'm a bit of a newbie with that

**Comment by al73rus** [comment]
Will double chec when i get home and try it once again on mobilr and pc

**Comment by ALERTua** [comment]
Solution:
- Open https://dash.cloudflare.com
- Change your Immich DNS Record to CNAME so it leads to your duckdns Immich counterpart
![image](https://github.com/user-attachments/assets/ae615f88-dcf4-4e6a-b9e7-153813853f9c)
- Listen for this address in your Nginx Proxy Manager and redirect it to the Immich instance
![image](https://github.com/user-attachments/assets/79f11429-9265-4cc5-b61f-4d2a03a41b6c)
![image](https://github.com/user-attachments/assets/b2fd8410-b0fc-4ec0-9968-87746d7e2c03)
